src/tests/miscellaneous/mock-adb/app/bin.js

```diff
@@ -84,6 +92,9 @@ async function main() {
     result.inputCommand = inputCommand;
     fs.writeFileSync(outputFile, JSON.stringify(result, null, '    '), { flag: 'a' });
 
+    const outputConfigFile = path.join(outputDir, 'mock_adb_config.json');
```

Comment:
Is there an easy to have 'mock_adb_config.json' defined in a single location, instead of defining it independently in 2 different files?